### PR TITLE
Added 'Add to AltStore' button to README

### PR DESCRIPTION
**Pull request by krinkuto11**
## Pull Request Description

Added a button that allows quick addition of the repo to AltStore from an iDevice. Due to the README not allowing 'altstore://' links, I had to do it through a link shortener. The non shortened URL is:
`altstore://source?url=https://raw.githubusercontent.com/bazzadazza72/fladder-develop/refs/heads/develop/altstore.json`

## Issue Being Fixed

No issue being fixed, just added the shortcut.

Issue Number: N/A

## Screenshots / Recordings
N/A

## Checklist

- [ ] If a new package was added, did you ensure it works for all supported platforms? Is the package also well maintained?
- [ ] Did you add localization for any text? If yes, did you sort the .arb file using ```arb_utils sort <INPUT_FILE>```?
- [ ] Check that any changes are related to the issue at hand.

### CODE REVIEW

**Comment by krinkuto11** [comment]
Sorry for the mess, noticed the link I used was wrong. Closed the issue and fixed the link. Now tested and working.

File: README.md [hunk 1]
```diff
@@ -82,7 +82,10 @@ The Android app is currently in testing you can help testing by joining this gro
 
 <a href='https://play.google.com/store/apps/details?id=nl.jknaapen.fladder&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1'><img alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png' width=250/></a>
 
-## How to Contribute
```
Comment (PartyDonut) [inline README.md:85]: Think you might have accidently remove this line 😃

File: README.md [hunk 2]
```diff
@@ -82,7 +82,10 @@ The Android app is currently in testing you can help testing by joining this gro
...
+### AltStore
+The iOS app can be currently installed through AltStore by adding the repo
```
Comment (PartyDonut) [inline README.md]: Small nitpick on my end, maybe change the text to something like this?

"The iOS app can be installed through AltStore by adding this repo"

**Comment by PartyDonut** [comment]
> Sorry for the mess, noticed the link I used was wrong. Closed the issue and fixed the link. Now tested and working.

That's fine thanks for taking adding the button.

One other thing the shorturl does it now link to this file?
`https://raw.githubusercontent.com/DonutWare/Fladder/refs/heads/develop/altstore.json`

You mentioned in your comment that the shorturl links to this url
`altstore://source?url=https://raw.githubusercontent.com/bazzadazza72/fladder-develop/refs/heads/develop/altstore.json`

File: README.md (hunk 1 above)
Comment (krinkuto11) [reply README.md:85]: Fixed

File: README.md (hunk 2 above)
Comment (krinkuto11) [reply README.md]: Definitely looks better. Changed.

**Comment by krinkuto11** [comment]
It resolves to the 'altstore://' URL so when opened through a phone it automatically opens AltStore. But yeah it is pointing to the file though the githubusercontent URL.

**Comment by PartyDonut** [comment]
The current url in the readme points to altstore://source?url=https://raw.githubusercontent.com/bazzadazza72/fladder-develop/refs/heads/develop/altstore.json

Specifically it points to bazzadazza72/fladder-develop/refs/heads/develop/altstore.json

Considering that we need to update it and to keep everything in one place it would be best to point it to thise repo. Like this.

altstore://source?url=https://raw.githubusercontent.com/DonutWare/Fladder/refs/heads/develop/altstore.json

**Comment by krinkuto11** [comment]
Should be changed now

**Review by PartyDonut — APPROVED** [review]

**Pull request merged**